features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2499,6 +2499,13 @@
                 <Origin>{{origin}}</Origin>
             {% endfor %}
         </FIDO2TrustedOrigins>
+
+        <!--
+            Time allocated for the users to authenticate
+            Time configurations are in milliseconds
```

Comment:
Fix formatting. Both comments should end with a full stop.

Reply:
Addressed with https://github.com/wso2/carbon-identity-framework/pull/3360/commits/debac20f6574d2b76a897bb1794f84d065703ed1